android/app/build.gradle

```diff
@@ -90,8 +90,8 @@ android {
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
         multiDexEnabled rootProject.ext.multiDexEnabled
-        versionCode 1001038200
-        versionName "1.3.82-0"
+        versionCode 1001038301
+        versionName "1.3.83-1"
```

Comment:
This is the current staging version though. if we push this to production, wont we skip one QA cycle?

Reply:
Not sure, but I think we should just close this and CP the one change into production if we need